api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceAuthService.java

```diff
@@ -114,15 +115,43 @@ public Map<String, FirecloudWorkspaceAccessEntry> getFirecloudWorkspaceAcls(
         fireCloudService.getWorkspaceAclAsService(workspaceNamespace, firecloudName));
   }
 
+  private void synchronizeOwnerBillingProjects(
```

Comment:
new helper method common to `update` and `patch`, extracted from the original `update`